plugins/Login/templates/login.twig

```diff
@@ -48,7 +48,7 @@
                             <input name="form_rememberme" type="checkbox" id="login_form_rememberme" value="1" tabindex="90"
                                    {% if form_data.form_rememberme.value %}checked="checked" {% endif %}/>
                             <label for="login_form_rememberme">{{ 'Login_RememberMe'|translate }}</label>
-                            <input class="submit btn" id='login_form_submit' type="submit" value="{{ 'Login_LogIn'|translate }}"
+                            <input class="submit btn" id="login_form_submit" type="submit" value="{{ 'Login_LogIn'|translate }}"
```

Comment:
no change needed just curious here as well... did the quote cause any issue? (just in case we need to pay attention to it in the future).

Reply:
No issue, and I don't even remember changing this file... :)